### ISSUE TITLE
Selection colour is hard to read.

### ISSUE THREAD

**Issue by kinda-raffy** [FourthGourd/macbeth-vscode#1]
The selection colour when highlighting words blends
too well with the background.

You can find this in the refactoring box.

**Comment by FourthGourd** [comment]
Yeah, I'll fix this during the next revision. All the colours need a few more variants, so once that's done I'll address all the highlighting problems.
